<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DodajPrzejazd</class>
 <widget class="QDialog" name="DodajPrzejazd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="numerParagonuLabel">
      <property name="text">
       <string>Numer paragonu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="numerParagonuLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="kierowcaLabel">
      <property name="text">
       <string>Kierowca</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="kierowcaBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="numerTelefonuKlientaLabel">
      <property name="text">
       <string>Numer telefonu klienta</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="numerTelefonucomboBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="stawkaLabel">
      <property name="text">
       <string>Stawka</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="stawkaComboBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="dataRozpoczeciaLabel">
      <property name="text">
       <string>Data rozpoczecia</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateTimeEdit" name="dataRozpoczeciaDateTimeEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="dataRozpoczeciaLabel_2">
      <property name="text">
       <string>Data zakończenia</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateTimeEdit" name="dataRozpoczeciaDateTimeEdit_2">
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="punktPoczTkowyLabel">
      <property name="text">
       <string>Punkt początkowy</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="punktPoczTkowyLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Punkt końcowy</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="punktKoncowyLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="dUgoTrasyLabel">
      <property name="text">
       <string>Długość trasy</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="dUgoTrasyDoubleSpinBox">
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DodajPrzejazd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DodajPrzejazd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
